pyo3-ffi/src/cpython/code.rs

```diff
@@ -39,7 +39,7 @@ pub struct PyCodeObject {
     pub co_extra: *mut c_void,
     #[cfg(Py_3_8)]
     pub co_opcache_map: *mut c_uchar,
-    #[cfg(Py_3_8)]
+    #[cfg(all(Py_3_8, not(Py_3_11)))]
```

Comment:
With https://github.com/PyO3/pyo3/pull/2409 I split this struct's definition, so this cfg now does nothing.

Reply:
👍 I'll tidy this up later.

Reply:
Done in #2432